### PR TITLE
Added a logo

### PR DESCRIPTION
**Pull request by CommanderStorm**
- [X] Added a logo
- [X] logo scales with screen-size

The logo is inline-svg, because then I can use the css-variables. If there is a way to do this without inline svgs I have not found it..

# Samples:
## Mobile
Screenshot 2021-01-10 173357

## Desktop
Screenshot 2021-01-10 173423

### CODE REVIEW

**Comment by kordianbruck** [comment]
Yea, you can use `xlink:href` I think to "inline" the thing and use the vars, [see this post](https://medium.com/@nikita.s2692/modify-svg-with-css-variables-7b4ce9f6eca1) for reference.

**Comment by CommanderStorm** [comment]
there is a problem with the new approach..
Screenshot 1
Screenshot 2021-01-12 232232

the parent does not recognise the childs height and thus does not scale approriately.

Possible Solution:

- [ ] change the svgs logo form a aproximately 4.12512891028:1 aspect-ratio to a ratio where width and height can be definded without whitespace (like 4:1 or similar)

Does anyone of you have a more promissing css-solution (my googling game is apparently not strong enough)?

File: index.php [hunk 1]
```diff
@@ -47,7 +50,13 @@ function constructLink($type, $link) {
 
 <body>
 <div id="main">
-    <h1 id="page-title"><i>RBGreater</i></h1>
+    <h1 id="page-title">
+        <a href="/" aria-label="RBGreater Logo">
+            <svg width="30rem" id="logo">
```
Comment (kordianbruck) [inline index.php:55]: Same, if you already specify it in CSS, why do it here again?

**Review by kordianbruck — APPROVED** [review]

**Comment by kordianbruck** [comment]
Thanks a lot!

**Pull request merged**